INSTALL

```diff
@@ -10,8 +10,10 @@ CellCC requires a few pieces of software to run:
  - perl (only tested with v5.16+, but may work with v5.8+)
  - OpenAFS, or something that provides an OpenAFS-like 'vos' command
  - AFS::Command
+ - On RHEL7|CentOS7 epel repo needs to be enabled.
```

Comment:
these aren't really requirements, but one way to satisfy the other requirements (some perl modules and remctl, I think?). I think this belongs in the "RPM" section